app/boards/shields/helix/helix_left.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+ 
+#include "helix.dtsi"
+
+&kscan0 {
+    col-gpios
+        = <&pro_micro_a 3 GPIO_ACTIVE_HIGH>
+        , <&pro_micro_a 2 GPIO_ACTIVE_HIGH>
+        , <&pro_micro_a 1 GPIO_ACTIVE_HIGH>
+        , <&pro_micro_a 0 GPIO_ACTIVE_HIGH>
+        , <&pro_micro_d 15 GPIO_ACTIVE_HIGH>
+        , <&pro_micro_d 14 GPIO_ACTIVE_HIGH>
+        , <&pro_micro_d 16 GPIO_ACTIVE_HIGH>
+    ;
```

Comment:
I've resolved this issue, and will include it in the next commit. Though, wouldn't it be better and easier to just replace this in all existing shield overlays? I don't think it would take too long to do. I wouldn't mind doing it myself.

Reply:
You're welcome to submit a PR for all shields, but please keep in mind that my passing comment was just my opinion and others (who wrote the existing shields) may disagree on this style (I honestly don't know)..

Reply:
This is the sort of debate a formatter would standardize.

Reply:
That's understandable. For now, I'll keep it equal to other existing shields for the sake of parity. May not be worth opening as an issue or editing otherwise until we get a formatter as you've discussed.